### PR TITLE
new custom fields for tableau enrollment view

### PR DESCRIPTION
**Pull request by GabyRangelB**
# Pull Request

## Summary & Motivation

[//]: # "When merged, this pull request will..."

## Self-review

- [x] Update **due date**, **assignee**, and **priority** on our
      [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

  _If this is a same-day request, please flag that in Slack_

- [x] <kbd>Format</kbd> has been run on all modified files

- [x] Ensure you are using the `union_dataset_join_clause()` macro for queries that employ any
      models from these datasets:
  - deanslist
  - edplan
  - iready
  - pearson
  - powerschool
  - renlearn
  - titan

## Troubleshooting

- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207923873298548

### CODE REVIEW

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql [hunk 1]
```diff
@@ -1,31 +1,92 @@
...
+            ) as rn,
+        from {{ ref("base_powerschool__student_enrollments") }}
+        where school_level = 'MS'
+    ),
+
+    dlm as (
+        select distinct student_number,
+        from {{ ref("int_students__graduation_path_codes") }}
+        where code = 'M'
+    )
```
[the remark this answers is not in the document]
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql:18]: Done!

File: src/dbt/kipptaf/models/students/intermediate/graduation_pathway_code_unpivot.sql
Comment (cbini) [inline src/dbt/kipptaf/models/students/intermediate/graduation_pathway_code_unpivot.sql:1]: rename `int_powerschool__graduation_pathway_code_unpivot` & update refs

File: src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql [hunk 2]
```diff
@@ -1,31 +1,91 @@
...
+        select
+            _dbt_source_relation,
+            student_number,
+            school_abbreviation as ms_attended,
+            row_number() over (
+                partition by student_number order by exitdate desc
+            ) as rn,
+        from {{ ref("base_powerschool__student_enrollments") }}
+        where school_level = 'MS'
+    )
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql:12]: note to self: can we move this further upstream?

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql (hunk 2 above)
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql:12]: > note to self: can we move this further upstream?

Should we dump it on student_filters?

Comment (cbini) [reply src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql:12]: maybe, but that's no upstream from this anyway. made an issue for myself

Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql:12]: Does this mean we can approve this PR with what we have for now? (pretty please!)

**Review by cbini — APPROVED** [review]

**Pull request merged**